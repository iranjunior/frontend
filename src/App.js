import React from 'react';
import { connect } from 'react-redux';

import Routes  from './Routes';


function App() {
  return (
          <Routes />
  
  );
}

export default connect(state => ({}))(App);
