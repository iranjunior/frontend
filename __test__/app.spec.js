import React from 'react';
import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';

// import configureStore from 'redux-mock-store';
import App from '../src/App';
/*
const mockStore = configureStore();
const initialState = [
  user: {
    username: 'usertest',
    email: 'usertest@gmail.com.br'
  }
] */
describe('Teste do componente Header', () => {
  it('renderizar corretamente', () => {
    const wrapper = shallow(<App />);
    // const wrapper = renderer.create(<App />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
