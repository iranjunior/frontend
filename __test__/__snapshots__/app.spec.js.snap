// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste do componente Header renderizar corretamente 1`] = `ShallowWrapper {}`;
